app.js

```diff
@@ -54,10 +55,12 @@ class App {
         const newConfIds = confIds.filter(id => !(id in this.conferenceStates));
         const removedConfIds = Object.keys(this.conferenceStates).filter(id => confIds.indexOf(id) === -1)
         newConfIds.forEach(newConfId => {
-            const dumpId = uuidv4();
+            const clientId = uuidv4();
             const confState = {
-                dumpId,
-                // TODO: use a set?
+                clientId,
+                confID: extractConferenceName(jvbJson, newConfId),
```

Comment:
So the field is called `confID` but the value is the conference name?  This is a bit confusing as those are 2 distinct things everywhere else.

Reply:
I find it a bit confusing as well, however this was the name used in jitsi-meet so I assumed it was the norm. We could potentially change the api.

Reply:
Ok, yeah I'd say if we can change it for the protocol that would probably be cleaner as I think it's more accurate.

Reply:
So conferenceName sound good?

Reply:
Yeah, or `confName`